### PR TITLE
Dutch translations

### PR DESCRIPTION
**Pull request by RenautMestdagh**
Updated "shared_album_section_people_action_leave" in en-US.json from "Remove user from album" to "Leave album".
Made nl-NL.json in line with en-US.json and added translations.

### CODE REVIEW

File: mobile/assets/i18n/nl-NL.json [hunk 1]
```diff
@@ -215,24 +236,38 @@
...
+  },
+  "map_cannot_get_user_location": "Kan locatie van de gebruiker niet ophalen",
+  "map_location_dialog_cancel": "Annuleren",
+  "map_location_dialog_yes": "Ja",
+  "map_location_picker_page_use_location": "Gebruik deze locatie",
+  "map_location_service_disabled_content": "Locatie service moet ingeschakeld zijn om assets van je huidige locatie weer te geven. Wil je het nu inschakelen?",
+  "map_location_service_disabled_title": "Locatie Service uitgeschakeld",
+  "map_no_assets_in_bounds": "Geen foto's in dit gebied",
+  "map_no_location_permission_content": "Locatie toestemming is nodig om assets van je huidige locatie weer te geven. Wil je het nu toestaan?",
+  "map_no_location_permission_title": "Locatien Toestemming geweigerd",
```
Comment (bo0tzz) [inline mobile/assets/i18n/nl-NL.json:253]:
```suggestion
  "map_no_location_permission_title": "Locatie Toestemming geweigerd",
```

**Comment by alextran1502** [comment]
We use Localizely for our translation need, please send me your email and I can send you the invitation link

File: mobile/assets/i18n/nl-NL.json [hunk 2]
```diff
@@ -261,20 +296,32 @@
...
+  "profile_drawer_client_out_of_date_minor": "Mobiele app is niet meer up-to-date. Werk bij naar de nieuwste onderversie.",
+  "profile_drawer_client_server_up_to_date": "Client en Server zijn up-to-date",
+  "profile_drawer_documentation": "Documentatie",
   "profile_drawer_github": "GitHub",
+  "profile_drawer_server_out_of_date_major": "Server is niet meer up-to-date. Werk bij naar de nieuwste hoofdversie.",
+  "profile_drawer_server_out_of_date_minor": "Server is niet meer up-to-date. Werk bij naar de nieuwste onderversie.",
   "profile_drawer_settings": "Instellingen",
   "profile_drawer_sign_out": "Uitloggen",
-  "profile_drawer_trash": "Trash",
+  "profile_drawer_trash": "Vuilbak",
```
Comment (bo0tzz) [inline mobile/assets/i18n/nl-NL.json:308]:
```suggestion
  "profile_drawer_trash": "Prullenbak",
```
Vuilbak is Flemish, I believe.

File: mobile/assets/i18n/nl-NL.json [hunk 3]
```diff
@@ -307,73 +356,116 @@
   "setting_notifications_single_progress_title": "Gedetailleerde informatie over back-ups op de achtergrond tonen",
   "setting_notifications_subtitle": "Voorkeuren voor meldingen beheren",
   "setting_notifications_title": "Meldingen",
-  "setting_notifications_total_progress_subtitle": "Algehele uploadvoortgang (voltooid/totaal aantal items)",
+  "setting_notifications_total_progress_subtitle": "Totale uploadvoortgang (voltooid/totaal aantal items)",
   "setting_notifications_total_progress_title": "Totale voortgang van achtergrondback-up tonen",
   "setting_pages_app_bar_settings": "Instellingen",
-  "settings_require_restart": "Start Immich opnieuw op om deze instelling toe te passen",
+  "settings_require_restart": "Please restart Immich to apply this setting",
```
Comment (bo0tzz) [inline mobile/assets/i18n/nl-NL.json:362]: Mistakenly added back the English version?

File: mobile/assets/i18n/nl-NL.json [hunk 4]
```diff
@@ -307,73 +356,116 @@
...
+  "shared_link_app_bar_title": "Gedeelde Links",
+  "shared_link_clipboard_copied_massage": "Gekopieerd naar klembord",
+  "shared_link_clipboard_text": "Link: {}\nWachtwoord: {}",
+  "shared_link_create_app_bar_title": "Link maken om te delen",
+  "shared_link_create_error": "Fout bij het maken van gedeelde link",
+  "shared_link_create_info": "Laat iedereen met de link de geselecteerde foto(s) zien",
+  "shared_link_create_submit_button": "Link maken",
+  "shared_link_edit_allow_download": "Openbare gebruiker toestaan om te downloaden",
+  "shared_link_edit_allow_upload": "Openbare gebruiker toestaan om te uploaden",
+  "shared_link_edit_app_bar_title": "Bewerkt link",
```
Comment (bo0tzz) [inline mobile/assets/i18n/nl-NL.json:388]:
```suggestion
  "shared_link_edit_app_bar_title": "Bewerk link",
```

File: mobile/assets/i18n/nl-NL.json [hunk 5]
```diff
@@ -307,73 +356,116 @@
...
+  "shared_link_expires_never": "Verloopt ∞",
+  "shared_link_info_chip_download": "Download",
+  "shared_link_info_chip_metadata": "EXIF",
+  "shared_link_info_chip_upload": "Upload",
+  "shared_link_manage_links": "Beheer Gedeelde links",
+  "share_done": "Klaar",
   "share_invite": "Uitnodigen voor album",
   "sharing_page_album": "Gedeelde albums",
-  "sharing_page_description": "Maak gedeelde albums om foto's en video's te delen met mensen in je netwerk.",
+  "sharing_page_description": "Create shared albums to share photos and videos with people in your network.",
```
Comment (bo0tzz) [inline mobile/assets/i18n/nl-NL.json:437]: English

File: mobile/assets/i18n/nl-NL.json [hunk 6]
```diff
@@ -307,73 +356,116 @@
...
   "theme_setting_three_stage_loading_title": "Laden in drie fasen inschakelen",
-  "translated_text_options": "Options",
-  "trash_page_delete": "Delete",
-  "trash_page_delete_all": "Delete All",
-  "trash_page_empty_trash_btn": "Empty trash",
-  "trash_page_empty_trash_dialog_content": "Do you want to empty your trashed assets? These items will be permanently removed from Immich",
+  "translated_text_options": "Opties",
+  "trash_page_delete": "Verwijder",
+  "trash_page_delete_all": "Verwijder Alle",
+  "trash_page_empty_trash_btn": "Leeg vuilbak",
```
Comment (bo0tzz) [inline mobile/assets/i18n/nl-NL.json:459]:
```suggestion
  "trash_page_empty_trash_btn": "Leeg prullenbak",
```

**Review by bo0tzz — CHANGES_REQUESTED** [review]
Thanks!

File: mobile/assets/i18n/nl-NL.json (hunk 2 above)
Comment (RenautMestdagh) [reply mobile/assets/i18n/nl-NL.json:308]: Yes indeed. I'm from belgium. Should there be a different file (nl-BE)?

**Comment by alextran1502** [comment]
Sent invitation over localizely.

**Pull request closed without merging**